### ISSUE TITLE
Derivation in nixpkgs

### ISSUE THREAD

**Issue by gador** [styx-static/styx#70]
Hi, 

I have noticed styx [didn't evaluate](https://hydra.nixos.org/job/nixpkgs/trunk/styx.x86_64-linux) since April anymore due to python2.7 in nixpkgs. After updating according to your derivation, it builds again, but takes *ages* to evaluate. Probably due to `nixpkgs` used in the new version.

Is there a reason `styx` is called [this way](https://github.com/styx-static/styx/blob/1d45ff744a34b09467dcd81adcab1737cb5dc357/src/styx.sh#L118)? 
Could you maybe chip in in https://github.com/NixOS/nixpkgs/pull/189123 ? I'd love to have `styx` be included in `nixpkgs` and work again with `nix-shell -p styx`.

Thanks for your time and effort!

**Comment by blaggacao** [comment]
Yeah, this intriguing excursiveness w.r.t. `nixpkgs` needs refactoring. It's on the backlog.
